lib/board.rb

```diff
@@ -0,0 +1,68 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells, :board, :token
+
+  turn_count = 0
+
+  def initialize
+    self.cells = Array.new(9, " ")
+    board
+  end
+
+
+
+def reset!
+  self.cells = Array.new(9, " ")
+end
+
+def display
+  puts " #{cells[0]}"" | ""#{cells[1]}"" | " "#{cells[2]} "
+  puts "-----------"
+  puts " #{cells[3]}"" | ""#{cells[4]}"" | " "#{cells[5]} "
+  puts "-----------"
+  puts " #{cells[6]}"" | ""#{cells[7]}"" | " "#{cells[8]} "
+end
+
+def position(input)
+  index = input.to_i - 1
+  cells[index]
+end
+
+def full?
+  if cells.any? { |cell| cell == " "}
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression.